### ISSUE TITLE
[Feature Request]table里有on-select-all的事件触发，有一些场景需要检测到on-select-all-cancel取消的时候触发，而on-selection-change我可能需要一些其他方法才知道我哪些取消了！

### ISSUE THREAD

**Issue by ddcouples** [iview/iview#2586]
### What problem does this feature solve?
当前table数据全不选的时候有回调函数触发

### What does the proposed API look like?
on-select-all-cancel

**Comment by MinimalistYing** [comment]
Agree!Really think the table should trigger some events like on-cancel-select-all.I can'y find a way to determine whether the behavior is deselect all or deselect a single row.

**Comment by icarusion** [comment]
https://github.com/iview/iview/commit/d483c54536a3d58763dd08edbec66b57de094367

**Comment by KendallMa** [comment]
maybe the return value should be cancelled value
